src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs

```diff
@@ -154,13 +228,43 @@ public async Task StatelessData_Put_RequestMissingBodyBadRequest()
 
             string requestUri = $"/{org}/{app}/v1/data?dataType=default";
 
-            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Put, requestUri);
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Post, requestUri);
 
             // Act
             HttpResponseMessage res = await client.SendAsync(httpRequestMessage);
 
             // Assert
             Assert.Equal(HttpStatusCode.BadRequest, res.StatusCode);
         }
+
+        [Fact]
+        public async Task StatelessData_Get_CalculationsRunAndDataReturned()
+        {
+            // Arrange
+            string org = "ttd";
+            string app = "dayplanner";
+
+            string token = PrincipalUtil.GetToken(1337);
+
+            HttpClient client = SetupUtil.GetTestClient(_factory, org, app);
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+
+            string requestUri = $"/{org}/{app}/v1/data?dataType=default";
+            
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, requestUri)
+            {
+            };
```

Comment:
Empty initialization block can be removed.

Reply:
done